tox_core/src/dht/server/mod.rs

```diff
@@ -2931,19 +2934,6 @@ mod tests {
         assert!(rx.collect::<Vec<_>>().await.is_empty());
     }
 
-    #[tokio::test]
-    async fn refresh_onion_key() {
```

Comment:
Why did you remove this test?

Reply:
Because the key is private in `XSalsa20Poly1305` struct, so we can't compare it anymore.